src/app/machines/components/MachineHeaderForms/MachineActionFormWrapper/CloneForm/CloneFormFields/CloneNetworkTable/CloneNetworkTable.test.tsx

```diff
@@ -47,43 +42,53 @@ describe("CloneNetworkTable", () => {
   });
 
   it("renders empty table if neither loading machine nor machine provided", () => {
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <CloneNetworkTable machine={null} selected={false} />
-      </Provider>
-    );
-    expect(wrapper.find("MainTable").prop("rows")).toStrictEqual([]);
-    expect(wrapper.find("Placeholder").exists()).toBe(false);
+    renderWithMockStore(<CloneNetworkTable machine={null} selected={false} />, {
+      state,
+    });
+    expect(screen.getAllByRole("row")).toHaveLength(1);
+    // expect(wrapper.find("MainTable").prop("rows")).toStrictEqual([]);
+    // expect(wrapper.find("Placeholder").exists()).toBe(false);
   });
 
   it("renders placeholder content while details are loading", () => {
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <CloneNetworkTable
-          loadingMachineDetails
-          machine={null}
-          selected={false}
-        />
-      </Provider>
+    renderWithMockStore(
+      <CloneNetworkTable
+        loadingMachineDetails
+        machine={null}
+        selected={false}
+      />,
+      {
+        state,
+      }
     );
-    expect(wrapper.find("Placeholder").exists()).toBe(true);
+    const rows = screen.getAllByRole("row");
+    rows.shift(); // Remove the table header row
```

Comment:
Could you use an alternative way to query table body rows only that doesn't involve mutating the array (as I already suggested in another PR).